### PR TITLE
Extend the BNC corpus reader with more tags.

### PR DESCRIPTION
**Pull request by dimazest**
BNC is now available at http://www.ota.ox.ac.uk/desc/2554

These are the changes I had to make to be able to read this version of the BNC.

### CODE REVIEW

File: nltk/corpus/reader/bnc.py [hunk 1]
```diff
@@ -16,11 +13,20 @@
...
     method.  For access to simple word lists and tagged word lists, use
     ``words()``, ``sents()``, ``tagged_words()``, and ``tagged_sents()``.
+
+    You can obtain the full version of the BNC corpus at
+    http://www.ota.ox.ac.uk/desc/2554
+
+    If you extracted the archive to a directory called `BNC`, then you can
+    instantiate the reder as::
+
+        BNCCorpusReader(root='BNC/Texts/', fileids=r'[A-K]/\w*/\w*\.xml')
```
Comment (kmike) [inline nltk/corpus/reader/bnc.py:28]: should we make this value for fielids default?

Comment (stevenbird) [inline nltk/corpus/reader/bnc.py:28]: Will this work across BNC versions?

Comment (dimazest) [inline nltk/corpus/reader/bnc.py:28]: i'm not sure. maybe it worth to leave it the documentation, so there is a reasonable hint for the parameter.

Comment (dimazest) [inline nltk/corpus/reader/bnc.py:28]: should i then remove the hint, or leave it as it is?

**Pull request merged**